### PR TITLE
(blog) Status update 2020-06-26

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

File: _posts/2020-06-26-status-update.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+We're already working on a improved fix which we hope to be able to share soon.
+As always we want to thank the community for their support and understanding and we hope to see you continuing your contributions.
+
+## Community Reports Updated
+
+The [community reports](https://puppetlabs.github.io/community_management/) are used by us to handle responding to all your fabulous contributions to the 82 repos we're responsible for.
+As part of her thesis, [Diana](https://github.com/daianamezdrea/) added a number of new reports and filters to these pages, and also has done a complete overhaul of the design.
+The reporting changes are already deployed, and the design changes will follow after her final exams at the start of next week.
+Please join us in wishing her all the best for the exams.
+We know you'll crush it, Diana!
```
Comment (DavidS) [inline _posts/2020-06-26-status-update.md:42]: FYI @daianamezdrea

File: _posts/2020-06-26-status-update.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## Ruby 2.7
+
+We've rolled out [Ruby 2.7 compatibility and testing](https://tickets.puppetlabs.com/browse/IAC-857) for all our tools over the last weeks.
+This includes new `-r2.7` versions of the [puppet-module-gems](https://github.com/puppetlabs/puppet-module-gems), as well as sundry fixes across the repos.
+The most common issue that needs addressing is disambiguating passing hashes vs keyword arguments, which [ruby 2.7 is getting picky about](https://rubyreferences.github.io/rubychanges/2.7.html#keyword-argument-related-changes).
+
+## Updating Bolt
+
+Thanks to the Bolt for fixing [an issue](https://github.com/puppetlabs/bolt/issues/1846) that was impacting litmus use.
```
Comment (sanfrancrisko) [inline _posts/2020-06-26-status-update.md:63]:
```suggestion
Thanks to the Bolt team for fixing [an issue](https://github.com/puppetlabs/bolt/issues/1846) that was impacting Litmus use.
```

**Comment by DavidS** [comment]
commited @sanfrancrisko 's suggestion

File: _posts/2020-06-26-status-update.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+The IAC team has been working on a lot of things over the past week.
+We want to share with our community some of the most interesting things.
+Here’s a brief overview of what we’ve been up to:
+
+## Module Releases from IAC Team
+
+To avoid the various infrastructure issues that have been plaguing our release verification, we've switched to manual test runs and have managed to release the following modules:
+- [puppetlabs/postgresql](https://github.com/puppetlabs/puppetlabs-postgresql) (6.6.0)
+- [puppetlabs/mysql](https://github.com/puppetlabs/puppetlabs-mysql) (10.6.0)
+- [puppetlabs/package](https://github.com/puppetlabs/puppetlabs-package) (1.2.0)
```
Comment (sanfrancrisko) [inline _posts/2020-06-26-status-update.md:27]:
```suggestion
- [puppetlabs/package](https://github.com/puppetlabs/puppetlabs-package) (1.2.0)
- [puppetlabs/vcsrepo](https://github.com/puppetlabs/puppetlabs-package) (3.1.1)
```

**Pull request merged**